### ISSUE TITLE
Pip index error

### ISSUE THREAD

**Issue by arenasys** [arenasys/qDiffusion#34]
```
GUI 2023-10-08 20:49:45.776689
Traceback (most recent call last):
  File "E:\AI\qDiffusion-master\source\main.py", line 175, in run
    raise RuntimeError("Failed to install: ", p, "\n", output)
RuntimeError: ('Failed to install: ', 'basicsr==1.4.2', '\n', "Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple\nCollecting basicsr==1.4.2\nUsing cached https://pypi.tuna.tsinghua.edu.cn/packages/86/41/00a6b000f222f0fa4c6d9e1d6dcc9811a374cabb8abb9d408b77de39648c/basicsr-1.4.2.tar.gz (172 kB)\nPreparing metadata (setup.py): started\nPreparing metadata (setup.py): finished with status 'done'\nCollecting addict (from basicsr==1.4.2)\nUsing cached https://pypi.tuna.tsinghua.edu.cn/packages/6a/00/b08f23b7d7e1e14ce01419a467b583edbb93c6cdb8654e54a9cc579cd61f/addict-2.4.0-py3-none-any.whl (3.8 kB)\nCollecting future (from basicsr==1.4.2)\nUsing cached future-0.18.3-py3-none-any.whl\nCollecting lmdb (from basicsr==1.4.2)\nUsing cached https://pypi.tuna.tsinghua.edu.cn/packages/66/05/21a93eed7ff800f7c3b0538eb12bde89660a44693624cd0e49141beccb8b/lmdb-1.4.1-cp310-cp310-win_amd64.whl (100 kB)\nRequirement already satisfied: numpy>=1.17 in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (1.24.1)\nCollecting opencv-python (from basicsr==1.4.2)\nUsing cached https://pypi.tuna.tsinghua.edu.cn/packages/38/d2/3e8c13ffc37ca5ebc6f382b242b44acb43eb489042e1728407ac3904e72f/opencv_python-4.8.1.78-cp37-abi3-win_amd64.whl (38.1 MB)\nRequirement already satisfied: Pillow in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (9.3.0)\nRequirement already satisfied: pyyaml in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (6.0.1)\nRequirement already satisfied: requests in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (2.28.1)\nRequirement already satisfied: scikit-image in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (0.22.0)\nRequirement already satisfied: scipy in e:\\ai\\qdiffusion-master\\venv\\lib\\site-packages (from basicsr==1.4.2) (1.11.3)\nINFO: pip is looking at multiple versions of basicsr to determine which version is compatible with other requirements. This could take a while.\nERROR: Could not find a version that satisfies the requirement tb-nightly (from basicsr) (from versions: none)\nERROR: No matching distribution found for tb-nightly\n")
```

**Comment by arenasys** [comment]
May be resolved by 5e707deb4973886bb39c8ec6f55b2000e2c6e006. Delete and redownload qDiffusion.

**Comment by darks0u1s** [comment]
issue fixed, able to install and run now, thanks!
